.github/workflows/build-deploy-pudl.yml

```diff
@@ -137,7 +137,7 @@ jobs:
             --container-env AWS_DEFAULT_REGION=${{ secrets.AWS_DEFAULT_REGION }} \
             --container-env AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }} \
             --container-env BUILD_ID=${{ env.BUILD_ID }} \
-            --container-env BUILD_REF=${{ env.BUILD_REF }} \
+            --container-env BUILD_REF=${{ github.ref_name }} \
```

Comment:
Huh, was this just a merge resolution oversight?

Reply:
Yeah, I think so.